pkg/metric/fuse_stat_collector.go

```diff
@@ -12,7 +13,7 @@ import (
 
 var (
 	fsClientPathPrefix      = "/host/var/run/"
-	fsClientTypeArray       = []string{"ossfs", "efc"}
+	fsClientTypeArray       = []string{"efc", "ossfs"}
```

Comment:
what's the difference?

Reply:
No difference in fact ... I tried to rewrite a collector for ossfs firstly, so I removed it from the list :/